tests/data/notebooks/outputs/ipynb_to_marimo/frozen_cell.py

```diff
@@ -0,0 +1,22 @@
+import marimo
+
+__generated_with = "0.15.2"
+app = marimo.App()
+
+
+@app.cell
+def _():
+    # This is an unfrozen cell. Works as usual.
+    print("I'm a regular cell so I run and print!")
+    return
+
+
+@app.cell
+def _():
```

Comment:
marimo has a `disabled=True` we could account for. We'll informat you if we add this, but probably not until someone asks